src/timedate/timedatectl.c

```diff
@@ -526,7 +526,7 @@ static int print_ntp_status_info(NTPStatusInfo *i) {
 
         r = table_print(table, NULL);
         if (r < 0)
-                log_error_errno(r, "Failed to show table: %m");
+               return table_log_show_error(r);
```

Comment:
Please use 8ch indentation

Reply:
ok，already fix it, tks，i will be more careful。

Reply:
oh，sorry, i ignore indeed. now all alread fix it. i will be careful next time.